### DISCUSSION TITLE
Do you guys know Nextcloud?

### DISCUSSION THREAD

**Discussion by grd** [helloSystem/hello#337] [Ideas]
Nextcloud [1] is a fork of Owncloud and is a German based cloud "storage system", but a lot more. There is also "Nextcloud desktop"[2] and even an app in iOS and Android. Personally I use Nextcloud and I like it, because it handles calendar, email, photo backup and well everything. The only problem is that it is a bit slow because it is web based. I want to discuss the "Nextcloud desktop",  because it is a C++ QML "app", that in Ubuntu is packaged as a snap. They also have a Windows and Mac app. So, why not here? 

You need to keep in mind that Nextcloud Desktop is *not* a "Windowing" app, it is *only* the population of your files to your desktop, most of the times it is populated in the folder "~/Nextcloud", and from there on all the files that are put in Nextcloud are populated automatically. From there on you need the web browser. The problem is probably that it needs to be run directly after startup of helloSystem, and also that "icon bars" need to be put. The icon bars are necessary because it indicates whether your server is running or updating files that you put into Nextcloud.

[1] https://nextcloud.com/
[2] https://github.com/nextcloud/desktop

**Comment by probonopd** [comment]
Hello @grd, it looks like it is available for FreeBSD:
https://www.freshports.org/deskutils/nextcloudclient/

Please try `sudo pkg install nextcloudclient` and see whether it works for you.

**Comment by grd** [comment]
I installed nextcloudclient. But when I try to run it, FreeBSD can't find it. So I tried it out with sudo but even then the answer was zero.

**Reply by grd** [reply]
Sorry. It works. You need to start "nextcloud" instead of "nextcloudclient".
